CMSgov/onemac
# OY2-11594 Regression Testing - User is able to create and submit duplicate SPAs and Waivers

Story: https://qmacbis.atlassian.net/browse/OY2-11594
Endpoint: https://d2ik0jdvbzlnqq.cloudfront.net

### Details

Old data is causing trouble.  While we wait for a proper data migration script, we should query ALL data sources for ID duplications because we cannot be guaranteed that the ID got saved in the OneTable yet.

### Changes
- modified packageExists to look at not only OneTable, but also SpaIds and changeRequests to make sure we're handling duplicate IDs properly.

### Implementation Notes
I originally modified the submit to also update SpaIds table, but Nagaraju requested that we try for as little impact on the testing as possible, so I feel like that means it would be better to not update submit.js and only expand pakageExists to look in all the places...

Plus... on feature branches, we seed the changeRequest table without modifying the supporting tables, so rather than fix that seed data at this time, better to check all tables, I feel.

### Test Plan
1. Login as State Submitter or State Admin user
2. Navigate to submission list dashboard
3. Note an existing SPA ID or Waiver number which were previously submitted (look for an old one!)
4. Create a new Medicaid SPA or CHIP SPA or Waiver by entering the noted SPA ID or Waiver ID which was submitted earlier
5. Submit it

Bad Behavior (should not happen)
User is able to submit SPA or Waiver successfully without displaying any error message saying that SPA ID or Waiver already exists

Expected Behavior
An error message should be displayed on SPA ID or Waiver ID field saying duplicate SPA ID or Waiver exists

NOTE: If the submit button updates get merged BEFORE this branch, the expected behavior is that the Submit Button STAYS INACTIVE (GRAY) and you cannot submit a duplicate, no matter the age/vintage of the other submission.
